flair/models/sequence_tagger_model.py

```diff
@@ -781,6 +781,14 @@ def _fetch_model(model_name) -> str:
         elif model_name in hu_model_map:
             model_path = cached_path(hu_model_map[model_name], cache_dir=cache_dir)
 
+            if model_name.startswith("hunflair"):
```

Comment:
Same here

Reply:
True. However, HunFlair2 will not be loaded as SequenceTaggerModel. I fix it anyway.